README.md

```diff
@@ -45,12 +47,21 @@ It is intended for use with editor/IDE integration.
 
 ## Installing
 
+### curl
```

Comment:
Can you change this to "Binary releases" please?

Reply:
Done!